### PR TITLE
smartmontools bump to 6.5

### PR DESCRIPTION
**Pull request by grueni**

### CODE REVIEW

**Review by AndWac — CHANGES_REQUESTED** [review]
1. Please move smartmontools to an appropriate category folder, eg. sysutils
2. Make use of $(WS_MAKE_RULES) for prep.mk, configure.mk and ips.mk in Makefile
3. Add REQUIRED_PACKAGES to Makefile

**Review by AndWac — CHANGES_REQUESTED** [review]
You still need to move the whole package into a component sub-folder, eg. components/sysutils.
We started to organize the packages some week ago.
Furthermore, when done you should squash the commits into a single one.

File: components/smartmontools/Makefile [hunk 1]
```diff
@@ -23,38 +23,42 @@
...
+COMPONENT_VERSION= 6.5
+COMPONENT_FMRI= system/storage/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION=	Applications/System Utilities
+COMPONENT_SRC= $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_PROJECT_URL= http://downloads.sourceforge.net/smartmontools
+COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:89e8bb080130bc6ce148573ba5bb91bfe30236b64b1b5bbca26515d4b5c945bc
+COMPONENT_ARCHIVE_URL= http://sourceforge.net/projects/smartmontools/files/smartmontools/${COMPONENT_VERSION}/$(COMPONENT_ARCHIVE)
+OMPONENT_LICENSE= GPLv2
+COMPONENT_LICENSE_FILE= $(COMPONENT_NAME).license
```
Comment (alarcher) [inline components/smartmontools/Makefile]: COMPONENT_LICENSE_FILE can be removed.

File: components/smartmontools/Makefile [hunk 2]
```diff
@@ -23,38 +23,42 @@
...
 
-include ../../make-rules/prep.mk
+include $(WS_MAKE_RULES)/prep.mk
 
 COMPONENT_PRE_CONFIGURE_ACTION = \
```
Comment (alarcher) [inline components/smartmontools/Makefile]: Put COMPONENT_PRE_CONFIGURE_ACTION after the includes.

File: components/smartmontools/Makefile [hunk 3]
```diff
@@ -23,38 +23,42 @@
...
-
+CONFIGURE_OPTIONS += --sysconfdir=/etc
+CONFIGURE_OPTIONS += --with-drivedbdir=yes
+CONFIGURE_OPTIONS += --with-savestates=/var/lib/smartmontools/smartd.
+CONFIGURE_OPTIONS += --with-attributelog=/var/lib/smartmontools/attrlog.
 
 build:		$(BUILD_32)
 
-test:		$(TEST_32)
+test:		$(NO_TEST)
```
Comment (alarcher) [inline components/smartmontools/Makefile]: Why disable the tests? Also the macros is NO_TESTS.

File: components/smartmontools/Makefile [hunk 4]
```diff
@@ -23,38 +23,42 @@
...
 
 install:	$(INSTALL_32)
 
-BUILD_PKG_DEPENDENCIES = $(BUILD_TOOLS)
-
-include ../../make-rules/depend.mk
+include $(WS_MAKE_RULES)/depend.mk
```
Comment (alarcher) [inline components/smartmontools/Makefile]: Line 59 can be removed.

File: components/smartmontools/smartmontools.p5m [hunk 5]
```diff
@@ -21,37 +21,42 @@
...
 
-license COPYING license='GPLv2'
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 dir path=var/lib/smartmontools
 
 file smartmontools.xml path=lib/svc/manifest/system/smartmontools.xml
 
 file path=etc/smartd.conf preserve=true
+file path=etc/smartd_warning.sh
```
Comment (alarcher) [inline components/smartmontools/smartmontools.p5m]: Is this shell script really supposed to be in /etc? Also if it is to be executed you need to append mode=0555

**Review by xen0l — APPROVED** [review]

**Comment by xen0l** [comment]
I'm fine with the changes and will be merging shortly. Thanks @grueni for the change, @AndWac and @alarcher for reviewing.

**Pull request merged**